config/recipes/elastic-agent/fleet-kubernetes-integration.yaml

```diff
@@ -79,8 +79,6 @@ spec:
       spec:
         serviceAccountName: fleet-server
         automountServiceAccountToken: true
-        securityContext:
-          runAsUser: 0
```

Comment:
I think it is still required for the container's CA bundle to be updated, see:
* https://github.com/elastic/cloud-on-k8s/blob/379c19a60f349ca31bd56837929e72e50a8a5b3e/pkg/controller/agent/pod.go#L360-L366
* https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-elastic-agent-fleet-known-limitations.html#k8s-elastic-agent-fleet-known-limitations